### PR TITLE
docs: add Mixcore CMS to showcase

### PR DESCRIPTION
**Pull request by Smilefounder**

### CODE REVIEW

**Comment by Smilefounder** [comment]
@Josh-Cena I have signed the CLA and added the screenshot, kindly review again.

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -1179,6 +1179,15 @@ const Users: User[] = [
...
     tags: ['design'],
   },
+  {
+    title: 'Mixcore CMS',
+    description:
+      'Fully Open Source ASP.Net Core / Dotnet Core CMS UI Toolkit',
+    preview: require('./showcase/mixcore.png'),
+    website: 'https://docs.mixcore.org/',
+    source: null,
+    tags: ['opensource', 'cms', 'opensource-cms', 'dotnet-core'],
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:1189]:
```suggestion
    tags: ['opensource'],
```

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -1179,6 +1179,15 @@ const Users: User[] = [
     source: null,
     tags: ['design'],
   },
+  {
+    title: 'Mixcore CMS',
+    description:
+      'Fully Open Source ASP.Net Core / Dotnet Core CMS UI Toolkit',
+    preview: require('./showcase/mixcore.png'),
+    website: 'https://docs.mixcore.org/',
+    source: null,
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:1188]:
```suggestion
    source: 'https://github.com/mixcore/docs/tree/master/website',
```

**Review by Josh-Cena — APPROVED** [review]
Thanks

**Pull request merged**